.github/workflows/lint-tests-release.yml

```diff
@@ -108,7 +108,7 @@ jobs:
         with:
           distribution: goreleaser
           version: latest
-          args: release --rm-dist
+          args: release --clean --debug
```

Comment:
should this be reverted?

Reply:
`--rm-dist` is deprecated in favor of `--clean`: https://goreleaser.com/deprecations/#-rm-dist.

I added `--debug` in thinking it may be helpful since we do have issues with goreleaser sometimes, but didn't get much from the debug output, so will revert that part of this change.

Reply:
Updated